core/src/dbs/options.rs

```diff
@@ -47,8 +46,6 @@ pub struct Options {
 	pub projections: bool,
 	/// The channel over which we send notifications
 	pub sender: Option<Sender<Notification>>,
-	/// Datastore capabilities
-	pub capabilities: Arc<Capabilities>,
```

Comment:
This property is not used anymore. We should probably stop using `pub` in favour of `pub (crate)` so we have a chance to identify unused properties. `Options` like `Context` are sensible parts of the code playing a direct role in query executions.